### ISSUE TITLE
csv

### ISSUE THREAD

**Issue by morganherlocker** [cutting-room-floor/distiller#4]
Do we want to support tabular formats like csv or xlsx?

**Comment by aaronlidman** [comment]
Yep, [csv is supported](https://github.com/mapbox/distiller/blob/master/test/basic.test.js#L24-L31). I'll put a list in the readme when we're more stable. Ideally we support everything supported in [mapnik-omnivore](https://github.com/mapbox/mapnik-omnivore).
